test-integration/README.md

```diff
@@ -5,13 +5,16 @@ Brief description on how to run integration tests.
 * OpenShift 4
 * Installed Operator
   * Operator has to watch different namespace then one it's installed in
+* Either valid context in `kubeconfig` file or set properties below.
 
 ## Running the tests
 * `mvn clean verify` to run all the tests
 * `mvn clean verify -Dit.test=${testName}` to run particular test
 * `mvn clean verify -Dit.test=org.infinispan.operator.*IT` to run functional tests that do not require cluster-admin access. User needs to have rights to manipulate with Infinispan CR objects however.
 * `mvn clean verify -Dit.test=org.infinispan.operator.installation.*IT` to run Operator installation tests that require cluster-admin access
 
+Tests retreive OpenShift context configuration from the current context in the `~/kube/config` file. You can override current context and fix the target environment using properties below.
```

Comment:
@Crumby we can provide `-Dxtf.openshift.admin.kubeconfig` property explicitly

Reply:
@dmvolod Yeah, I didn't wanted to include all the possibilities as there is really lot of them. Best would be to point to xtf framework documentation but it's not greatly documented there either

Reply:
Yes, @Crumby, would be nice and choose a section with connection options directly